### ISSUE TITLE
MathJax out of the box

### ISSUE THREAD

**Issue by ehaydenr** [digitalcraftsman/hugo-icarus-theme#26]
I noticed on the [intro page](http://themes.gohugo.io/theme/hugo-icarus/2015/10/10/introducing-icarus-and-its-features) that the MathJax isn't compiling. I am a new user to this theme and would like to know if this feature is broken or if I am missing something.

**Comment by digitalcraftsman** [comment]
Hi @ehaydenr,

the instructions should have been a bit more detailed. If I remember it right it worked as expected but later I noticed that the rendering intererred with the syntax highlighter (also written in JS). This messed up the content rendering.

In short: I've updated the instruction (see 87d8b91b87f7e85d4a9bef48181579299a3a422f). MathML works as expected out of the box. LaTeX code needs to be wrapped with `$$` for the rendering. The usage of a formula within a sentence (inline) requires only a single `$`.

Cheers,
Digitalcraftsman
